### PR TITLE
AS/meta application

### PR DESCRIPTION
**Pull request by magnearun**
## What

Add Meta application (application to apply for usage of the application system)

## Why

So someone from island.is can go over requests to use the application system

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against master before asking for a review

### CODE REVIEW

**Review by arnifreyrs — DISMISSED** [review]
Awesome!

File: libs/island-ui/core/src/lib/Input/Input.tsx [hunk 1]
```diff
@@ -35,6 +35,7 @@ interface InputProps extends InputComponentProps {
   tooltip?: string
   backgroundColor?: InputBackgroundColor
   textarea?: boolean
+  type?: 'text' | 'number' | 'email' | 'tel'
```
Comment (jeremybarbet) [inline libs/island-ui/core/src/lib/Input/Input.tsx:38]: Unused?

**Review by darrikonn — APPROVED** [review]

**Review by jeremybarbet — APPROVED** [review]
LGTM

File: libs/application/templates/meta-application/src/lib/ApplicationTemplate.ts [hunk 2]
```diff
@@ -0,0 +1,161 @@
+import {
+  ApplicationTemplate,
+  ApplicationTypes,
+  ApplicationContext,
+  ApplicationRole,
+  ApplicationStateSchema,
+} from '@island.is/application/core'
+import * as z from 'zod'
+
+const nationalIdRegex = /([0-9]){6}-?([0-9]){4}/
```
[the remark this answers is not in the document]
Comment (magnearun) [reply libs/application/templates/meta-application/src/lib/ApplicationTemplate.ts]: fixed

File: libs/island-ui/core/src/lib/Input/Input.tsx (hunk 1 above)
Comment (magnearun) [reply libs/island-ui/core/src/lib/Input/Input.tsx]: yes, it was going with the spread of ...inputProps. I moved it to the InputComponentProps and took it out of the spread

**Pull request merged**